docs/_static/auth/iam.yaml

```diff
@@ -607,6 +607,128 @@ paths:
                 "$ref": "#/components/schemas/VerifiableCredential"
         default:
           $ref: '../common/error_response.yaml'
+  /internal/auth/v2/{did}/request-credential:
+    post:
+      operationId: requestOid4vciCredentialIssuance
+      summary: Start the Oid4VCI authorization flow.
+      description: |
+        Initiates an Oid4VCI flow to request an VC from a Credential Issuer.
+
+        error returns:
+        * 400 - one of the parameters has the wrong format or an OAuth error occurred
+        * 424 - the issuer does not fulfill the right requirements to issue the requested VC(s)
+        * 412 - the organization wallet does not contain the correct credentials
+        * 503 - the authorizer could not be reached or returned an error
```

Comment:
maybe you want to put this under the 424s, 503 are usually the result of gateway to backend communication.

Reply:
Removed the 503 error state.